commons/src/main/java/fr/free/nrw/commons/MediaWikiImageView.java

```diff
@@ -151,7 +151,7 @@ private void loadImageIfNecessary(final boolean isInLayoutPass, final boolean tr
         // from the network.
         ImageContainer newContainer = mImageLoader.get(mUrl,
                 new ImageListener() {
-                    @Override
```

Comment:
My language level is set to SDK default (8) as shown in that thread. My project SDK is different from theirs though as it is an Android project. But changing the project SDK does not seem to change the error.